tests/library/proxy.spec.ts

```diff
@@ -350,7 +350,7 @@ it('should use http proxy for websocket requests', async ({ browserName, browser
 
   // WebKit does not use CONNECT for websockets, but other browsers do.
   if (browserName === 'webkit')
-    expect(proxyServer.wsUrls).toContain('ws://fake-localhost-127-0-0-1.nip.io:1337/ws');
+    expect(proxyServer.wsUrls).toContain(isWindows ? '/ws' : 'ws://fake-localhost-127-0-0-1.nip.io:1337/ws');
```

Comment:
is this an unrelated test re-basing or caused by this PR?

Reply:
This is unrelated, the test fails on windows on the bots, so I included the drive-by fix.